### DISCUSSION TITLE
Is the build task expected to work with Fresh 2 alpha?

### DISCUSSION THREAD

**Discussion by kylec32** [freshframework/fresh#2662] [General]
I know, I'm a glutton for punishment but based on [this comment](https://github.com/denoland/fresh/issues/2363#issuecomment-2350955302) I figured with a brand new project I would try starting with the latest alpha of Fresh 2. Basically everything has worked which is awesome! I went to build the project and I'm presented with:

```
error: Uncaught (in promise) TypeError: Deno.resources is not a function
    const children = Object.values(Deno.resources())
                                        ^
    at bundleJs (https://jsr.io/@fresh/core/2.0.0-alpha.20/src/dev/esbuild.ts:135:41)
    at eventLoopTick (ext:core/01_core.js:175:7)
    at async Builder.#build (https://jsr.io/@fresh/core/2.0.0-alpha.20/src/dev/builder.ts:174:20)
    at async Builder.build (https://jsr.io/@fresh/core/2.0.0-alpha.20/src/dev/builder.ts:103:12)
    at async file:///<project_dir>/dev.ts:69:3
```
I also get the same thing with a fresh/newly init-ed project. I take that as it means that the build task isn't currently working? I get it, its an alpha so if it isn't expected to work that's fine. Just wanted to make sure I wasn't messing something up. There is even a "Fresh 2" framework preset on Deno Deploy so it kind of felt like I would have expected it to work. If there is additional information I can provide to help the development I'm happy to do that as well. (I realize Fresh 2 development has been paused to help with the Deno 2 release but that seems real close as well)

**Comment by marvinhagemeister** [comment]
Starting with Fresh 2.0.0-alpha.21 Deno 2 is supported. To resolve this error update Fresh.

~~Fresh 2 doesn't support Deno 2 rc yet. It will once Deno 2 is out. To resolve this run `deno upgrade` to downgrade to the latest stable Deno release.~~
